Distinguish between aggregate & device properties

cjsha opened the pull request:
Resolve #171

![image](https://github.com/user-attachments/assets/09c5303a-7f5e-4e81-8772-7634879ac6b7)
The space between the header and the badge for a hyperlink to appear when hovering the cursor there.

Something I don't like from this initial commit is that I have red and green badges to distinguish between acquisition and device properties. I'm worried that there is an instinctual heuristic that red = bad and green = good that I don't want to evoke from people skimming our docs. I can use different colors, but we might want to reserve two colors to for badges when attempting a solution for #98.

jonnew commented:
This looks very good. Better than original idea. 

I think maybe the word "aggregate" is a bit scary. Which is why Cris originally came up with "MultiDeviceFactory". So I might want to change that.  Also "aggregate" is an adjective, noun, and verb! So its confusing in this sense as well.  Maybe "Device Group"

Another note -- is there a reason these are lowercased? I think they should be uppercase: "Device", "Acquisition", etc.

re Red = bad: yeah thats true. I think we just need to design some more colors. For instance, orange and green like is used in [RP2040 docs](https://www.raspberrypi.com/documentation/pico-sdk/hardware.html#group_hardware_gpio_1gaf91b38a4d4e96c343bd16927c34936e0) that I happen to be looking at:

<img width="476" alt="image" src="https://github.com/user-attachments/assets/19580c3a-e267-4264-8e80-311ff170cffe" />

cjsha commented:
I was originally limiting myself to [these colors](https://getbootstrap.com/docs/5.3/components/alerts/#examples) but I saw how bootstrap generates an alert's background color, border color, and text color (and their dark versions) from a given base color so I expanded the available color palette. For example:
![image](https://github.com/user-attachments/assets/7f73f57a-0e5d-4ee4-b4c6-a9285b28943b) ![image](https://github.com/user-attachments/assets/a94656cd-c8b4-4978-9e15-803a571eb911)

Those are all the badges I foresee being used in these docs and my proposal for their colors. Currently "Configuration" is yellow, but I think opposing labels should oppose on the primary/secondary color wheel so I switched it to orange in the above screenshot. Fixed-rate and variable-rate refer to "event-based" and "streaming" data operators in #98.

jonnew commented:
These badges look great. @cjsha and I discussed the Fixed and Variable rate badges IRL and realized that it might be difficult to maintain since they would need to be manually added or we would need to make up an attribute for data operators and apply that in the source code. Both sound clunky. Additionally, at the end of the day, from the API's perspective and in Bonsai, there is only variable rate data because everything is asynchronous. So, lets couch that for now. 

In the meantime, I would replace the red device color with the nice purple from the the proposed variable rate tag.

jonnew commented on api/configure.md:
```diff
@@ -1,9 +1,13 @@
-configure ONIX hardware hubs. These are known as aggregate configuration operators because they configure an aggregation
-of devices (also referred to as a hub) on a given headstage, miniscope, breakout board, etc..
\ No newline at end of file
+configure ONIX hardware hubs. These are known as device group configuration operators because they configure an aggregation
```
"configure an aggregation" -> configure a group

jonnew commented on api/configure.md:
```diff
@@ -1,9 +1,13 @@
-of devices (also referred to as a hub) on a given headstage, miniscope, breakout board, etc..
\ No newline at end of file
+configure ONIX hardware hubs. These are known as device group configuration operators because they configure an aggregation
+of devices (also referred to as a hardware hub) on a given headstage, miniscope, breakout board, etc..
```
dont use hub, its a specification term.

jonnew commented on api/configure.md:
```diff
@@ -1,9 +1,13 @@
+configure ONIX hardware hubs. These are known as device group configuration operators because they configure an aggregation
+of devices (also referred to as a hardware hub) on a given headstage, miniscope, breakout board, etc..
+
+Aggregate configuration operators are simply referred to as "Configuration Operators" in the
```
Device Group

jonnew commented on api/configure.md:
```diff
@@ -1,9 +1,13 @@
+of devices (also referred to as a hardware hub) on a given headstage, miniscope, breakout board, etc..
+
+Aggregate configuration operators are simply referred to as "Configuration Operators" in the
+[Reference](xref:OpenEphys.Onix1) TOC for concision and simplicity as they are recommended starting-point operators for
```
TOC -> Table of contents

jonnew commented on articles/getting-started/property-categories.md:
edited directly by @jonnew

jonnew reviewed (changes requested):
The colors for the tags dont show up in duck duck go browser

cjsha commented:
i tried ddg on my machine without changing anything and the badge colors show. if you update, does it work? the problem _might_ be something else.

![image](https://github.com/user-attachments/assets/516f920c-1ffb-455d-a55f-a8054e000e9f)
